nexus/reconfigurator/planning/src/system.rs

```diff
@@ -446,7 +453,7 @@ impl Sled {
         let serial = format!("serial{}", unique);
         let revision = 0;
         let mut zpool_rng =
-            TypedUuidRng::from_seed("SystemSimultatedSled", "ZpoolUuid");
+            TypedUuidRng::from_seed("SystemSimultatedSled-ZpoolUuid", sled_id);
```

Comment:
Ah good catch. Though what about:

```suggestion
            TypedUuidRng::from_seed("SystemSimultatedSled", (sled_id, "ZpoolUuid"));
```